cmd/commands/runtime.go

```diff
@@ -819,40 +829,48 @@ func NewRuntimeUninstallCommand() *cobra.Command {
 }
 
 func RunRuntimeUninstall(ctx context.Context, opts *RuntimeUninstallOptions) error {
+	defer printSummaryToUser()
+
 	// check whether the runtime exists
 	if !opts.SkipChecks {
 		_, err := cfConfig.NewClient().V2().Runtime().Get(ctx, opts.RuntimeName)
+		appendLogToSummary("Checking if runtime exists", err)
 		if err != nil {
-			log.G(ctx).Warn("you can attempt to uninstall again with the \"--skip-checks\" flag")
+			summaryArr = append(summaryArr, summaryLog{"you can attempt to uninstall again with the \"--skip-checks\" flag", Info})
 			return err
 		}
 	}
 
 	log.G(ctx).Infof("Uninstalling runtime '%s'", opts.RuntimeName)
 
-	if err := apcmd.RunRepoUninstall(ctx, &apcmd.RepoUninstallOptions{
+	err := apcmd.RunRepoUninstall(ctx, &apcmd.RepoUninstallOptions{
 		Namespace:    opts.RuntimeName,
 		Timeout:      opts.Timeout,
 		CloneOptions: opts.CloneOpts,
 		KubeFactory:  opts.KubeFactory,
 		Force:        opts.Force,
 		FastExit:     opts.FastExit,
-	}); err != nil {
+	})
+	appendLogToSummary("Uninstalling repo", err)
+	if err != nil {
 		if !opts.Force {
-			log.G(ctx).Warn("you can attempt to uninstall again with the \"--force\" flag")
+			summaryArr = append(summaryArr, summaryLog{"you can attempt to uninstall again with the \"--force\" flag", Info})
 			return err
 		}
 	}
 
-	if err := deleteRuntimeFromPlatform(ctx, opts); err != nil {
+	err = deleteRuntimeFromPlatform(ctx, opts)
+	appendLogToSummary("Deleting runtime from platform", err)
+	if err != nil {
 		return fmt.Errorf("failed to delete runtime from the platform: %w", err)
 	}
 
 	if cfConfig.GetCurrentContext().DefaultRuntime == opts.RuntimeName {
 		cfConfig.GetCurrentContext().DefaultRuntime = ""
 	}
 
-	log.G(ctx).Infof("Done uninstalling runtime '%s'", opts.RuntimeName)
+	installationDoneStr := fmt.Sprintf("Done uninstalling runtime '%s'", opts.RuntimeName)
```

Comment:
the variable name is wrong

Reply:
what's wrong with it?

Reply:
it's not installation, it's uninstallation